docs/_static/spec/rasa.yml

```diff
@@ -296,6 +296,57 @@ paths:
         500:
           $ref: '#/components/responses/500ServerError'
 
+  /conversations/{conversation_id}/inject-intent:
+    post:
+      security:
+        - TokenAuth: []
+        - JWT: []
+      operationId: injectConversationIntent
+      tags:
+        - Tracker
+      summary: Inject an intent into a conversation
+      description: >-
+        Sends a message with the specified intent and entities in place
+        of the user, then predicts and executes a response action.
+        Any responses sent by the executed action will be forwarded
+        to the channel specified in the output_channel parameter.
+        If no output channel is specified, any messages that should be
+        sent to the user will be included in the response of this endpoint.
+      parameters:
+        - $ref: '#/components/parameters/conversation_id'
+        - $ref: '#/components/parameters/include_events'
```

Comment:
Would be good to quickly describe this parameter as well

Reply:
I just copied this from the `execute` event. The parameter descriptions are at the end of the file and should be automatically displayed in the doc. Isn't that enough?

Reply:
Ah I see, yep as the description is already inside that component